app/client/components/accountoverview/accountOverview.tsx

```diff
@@ -68,28 +86,43 @@ const AccountOverviewRenderer = (apiResponse: MembersDataApiItem[]) => {
       />
 
       {Object.entries(mmaCategoryToProductDetails).map(
-        ([mmaCategory, productDetails]) => {
+        ([mmaCategory, products]) => {
           const groupedProductType =
             GROUPED_PRODUCT_TYPES[mmaCategory as GroupedProductTypeKeys];
           return (
-            productDetails.length > 0 && (
+            products.length > 0 && (
               <React.Fragment key={mmaCategory}>
                 <h2 css={subHeadingCss}>
                   My {groupedProductType.groupFriendlyName}
                 </h2>
-                {productDetails.map(productDetail => (
-                  <AccountOverviewCard
-                    key={productDetail.subscription.subscriptionId}
-                    productDetail={productDetail}
-                  />
-                ))}
-                {productDetails.some(productDetail =>
-                  isCancelled(productDetail.subscription)
-                ) && (
-                  <SupportTheGuardianSection
-                    {...groupedProductType.supportTheGuardianSectionProps}
-                  />
-                )}
+                {products.map(product => {
+                  return isCancelledProduct(product) ? (
+                    <AccountOverviewCancelledCard
+                      key={product.subscription.subscriptionId}
+                      product={product as CancelledProductDetail}
+                    />
+                  ) : (
+                    <AccountOverviewCard
+                      key={product.subscription.subscriptionId}
+                      productDetail={product as ProductDetail}
+                    />
+                  );
+                })}
+                {(groupedProductType.groupFriendlyName === "membership" ||
+                  groupedProductType.groupFriendlyName === "contribution") &&
+                  products.some(product => {
+                    if (isCancelledProduct(product)) {
+                      return true;
+                    } else {
+                      return isCancelled(
```

Comment:
Having now both `isCancelledProduct` and `isCancelled` seems a bit confusing. Maybe `isCancelled` should be renamed to indicate this is not fully ended product, say,`cancellationIsPending`

Reply:
yep .. I'm going to factor out `isCancelledProduct` and use the separation of the active and cancelled products from the respective api calls to distinguish them